### PR TITLE
Feature/v1-003: Configure theme for Client

### PR DESCRIPTION
**Pull request by hauteballe**
Overview: 
1. Configured the common theme of the application
2. Created shared components (Button and Input)
3. Implemented app menu and header

### CODE REVIEW

File: frontend/src/components/layout/UserDropDown/UserDropDown.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  userUnit,
+  userDepartment,
+  userGroup,
+  avatarUrl,
+  isOpen,
+  onClick,
+  anchorEl,
+}) => (
+  <Box>
+    <UserInfo onClick={(event: React.MouseEvent<HTMLElement>) => onClick(event)}>
```
Comment (k-krasilnikova) [inline frontend/src/components/layout/UserDropDown/UserDropDown.tsx:39]: No need to use an arrow function here, the same:
```suggestion
    <UserInfo onClick={onClick}>
```

File: frontend/src/components/layout/UserDropDown/UserDropDownContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+const UserDropDown: React.FC<User> = ({
+  userName,
+  userRole,
+  userUnit,
+  userDepartment,
+  userGroup,
+  avatarUrl,
+}) => {
+  const [isDropdownOpen, setDropdownState] = React.useState(false);
+  const [anchorEl, setAnchorEl] = React.useState<HTMLElement | null>(null);
```
Comment (k-krasilnikova) [inline frontend/src/components/layout/UserDropDown/UserDropDownContainer.tsx:15]: Replace it with:
```suggestion
  const [isDropdownOpen, setDropdownState] = useState(false);
  const [anchorEl, setAnchorEl] = useState<HTMLElement | null>(null);
```
and
`import { useState } from 'react';`

File: frontend/src/pages/Layout.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+
+import PATHS from '../constants/routes';
+import Header from '../components/Layout/Header';
+import Menu from '../components/Layout/Menu';
```
Comment (k-krasilnikova) [inline frontend/src/pages/Layout.tsx:6]: absolute paths?

**Review by k-krasilnikova — CHANGES_REQUESTED** [review]
@Julia-Maliutsina @hauteballe fix please all the comments

**Review by Julia-Maliutsina — DISMISSED** [review]

**Review by k-krasilnikova — APPROVED** [review]

**Review by Julia-Maliutsina — APPROVED** [review]

**Pull request merged**